LibreNMS/OS/Aix.php

```diff
@@ -35,4 +35,21 @@ public function discoverOS(Device $device): void
         // don't support server hardware detection or extends
         $this->discoverYamlOS($device);
     }
+
```

Comment:
How come this is commented out?

Reply:
Likely I did it in PHP, then added yaml, then forgot about the PHP.  I'll remove this.